dspace/solr/search/conf/schema.xml

```diff
@@ -283,6 +283,7 @@
     <!-- used by the DSpace Discovery Solr Indexer to track the last time a document was indexed -->
     <field name="SolrIndexer.lastIndexed" type="date" indexed="true" stored="true" default="NOW" multiValued="false" omitNorms="true" />
     <field name="lastModified" type="date" indexed="true" stored="true" default="NOW" multiValued="false" omitNorms="true" />
+     <field name="lastModified_dt" type="date" indexed="true" stored="true" default="NOW" multiValued="false" omitNorms="true" docValues="true"/>
```

Comment:
Could we add a comment as to why this is different from "lastModified"?  Why do we need two indexed "date" fields with the same value here?

Reply:
after further checking we deleted the lastModified_dt field, currently, we use lastModified only